src/components/Product/SessionReplay/index.tsx

```diff
@@ -104,7 +104,7 @@ const subfeatures = [
     },
     {
         title: 'Web or mobile session recording',
-        description: 'Web or Android (beta) available',
+        description: 'Web, iOS, and Android',
```

Comment:
```suggestion
        description: 'Web, Android, and iOS',
```

just to match the sequence: Watch users interacting with your app or website. Available for web, Android, and iOS.

Reply:
should we mention React Native (beta) here?

Reply:
```suggestion
        description: 'Watch users on your app or website. Available for web, Android, and iOS.',
```